examples/meteor/v1beta1/meteor-controller.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Are you sure you want to add v1beta1 versions of these files? They may have a rather short lifecycle.